packages/plugins/rubygem-mqtt/rubygem-mqtt.spec

```diff
@@ -0,0 +1,93 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
+%global gem_name mqtt
+%global plugin_name mqtt
+%global foreman_min_version 3.0
```

Comment:
which template did you use? it's not a foreman plugin, so I think the `scl` template is the correct one.

Reply:
I used the plugin one, I'll do it again